.vscode/settings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "editor.rulers": [
+    80
+  ],
+  "editor.tabSize": 2,
+  "javascript.validate.enable": true,
+  "search.exclude": {
+    "**/node_modules": true,
+    "**/graphql": true,
+    ".cache/**": true
```

Comment:
Can we make this relevant to this project? No need for `graphql` or `cache` here.